torchao/quantization/quant_api.py

```diff
@@ -1434,18 +1434,21 @@ def _float8_weight_only_transform(
 _fp8_granularities = Union[PerTensor, PerRow]
 
 
-# Validate and process granularity input
 def _normalize_granularity(
     granularity: Optional[
-        Union[_fp8_granularities, Tuple[_fp8_granularities, _fp8_granularities]]
+        Union[
+            _fp8_granularities,
```

Comment:
general comment/question not specific to this PR: is it conventional to have python type aliases in snake case like this? seems odd to me, I expected camel case

Reply:
yeah good call changed